boxoffice/views/item_collection.py

```diff
@@ -29,7 +29,7 @@ def jsonify_item(item):
 
 def jsonify_category(category):
     category_items = []
-    for item in category.items:
+    for item in Item.query.filter(Item.category == category).order_by('seq ASC').all():
```

Comment:
`ASC` is implicit. Don't need to specify it. Also, since this is a view, add a class method to `Item` that accepts the `category` parameter and returns the query. Don't use `.all()`. Queries are iterable by default, `all()` just casts the iterator into a list.